lib/how.md

```diff
@@ -0,0 +1,35 @@
+## Config
+
+follow the `_example-config.yml` to setup a config file
+
+```
+sensor: {landsat8 or sentinel2}
+scenes:
+  - loc: {where the disaster happened}
+    pre:
+     - {sceneID for pre-disaster scene}
+     ...
+    post:
+     - {sceneID for post-disaster image}
+   ...
+```
+
+edit create a .env file with these two environmental variables
+
+```
+# this needs to be have upload priviledges. see: https://www.mapbox.com/help/create-api-access-token/
+MAPBOX_KEY='mapbox.api.key'
```

Comment:
I'd just use the `MAPBOX_ACCOUNT` and `MAPBOX_ACCESS_TOKEN` varnames that the Mapbox CLI expects in the end, right?